src/modules/gameDevs/gameDevWebhookHandler.ts

```diff
@@ -10,3 +18,33 @@ export const getAchievementsThruWebhookHandler: Lifecycle.Method = async (reques
   // }
   return h.response({ achievements }).code(200);
 };
+
+export const postLeaderboardResultHandler: Lifecycle.Method = async (request, h) => {
+  const { gameType } = request.pre.webhookValidation;
+  const payload = request.pre.webhookPayload as LeaderboardResultsCreationAttributes;
```

Comment:
payload is coming from the webhook payload parser as we mentioned on the call.
the schema validation occurs inside that middleware